### PR TITLE
Update wording and link in evolution.md

### PR DESCRIPTION
**Pull request by lamtrinhdev**
+ Update the wording from `other API` to `other APIs`.
+ Update the link of `swift-evolution`.

### CODE REVIEW

**Comment by lamtrinhdev** [comment]
Dear @parkera ,

Sorry for tagging you (because I see you are the last change of this file). I have a minor update for wording. Please help me to review it.

Thanks,
Lam

**Comment by jmschonfeld** [comment]
@swift-ci please test

File: Evolution.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Foundation Evolution Process
 
-All new public types, functions, and other API must go through an API review process. Foundation uses an API process inspired by the [Swift Evolution review process](https://github.com/apple/swift-evolution/blob/main/process.md).
+All new public types, functions, and other APIs must go through an API review process. Foundation uses an API process inspired by the [Swift Evolution review process](https://github.com/swiftlang/swift-evolution/blob/main/process.md).
```
Comment (parkera) [inline Evolution.md:3]: I’ve always considered API to be the same “word”, singular or plural. However I am unsure if we have any consistency around this in the READMEs here, or other documentation.

Comment (lamtrinhdev) [reply Evolution.md:3]: Dear @parkera ,

Thank you for your feedback. As I refer at https://developer.apple.com and https://www.swift.org . They are using APIs.

+ https://developer.apple.com/documentation
` Browse new and updated documentation for technologies, frameworks, and APIs.`

+ https://www.swift.org/documentation/
`Swift has support for bidirectional interoperability with C++. A great variety of C++ APIs can be called directly from Swift, and select Swift APIs can be used from C++. `

+ https://www.swift.org/documentation/api-design-guidelines/
`Delivering a clear, consistent developer experience when writing Swift code is largely defined by the names and idioms that appear in APIs.`

+ https://www.swift.org/documentation/standard-library/
`The [Standard Library Preview package](https://github.com/apple/swift-standard-library-preview/) provides early access to new additions to the standard library. When new standard library APIs that can be implemented as a standalone library are accepted through the Swift Evolution process`

This is just a minor update. If I am wrong, please support me to close this ticket.

Thanks,
Lam

Comment (lamtrinhdev) [reply Evolution.md:3]: Dear @parkera ,

How do you feel about this? If this does not make sense, we will close the ticket.

Thanks,
Lam

**Review by parkera — APPROVED** [review]

**Pull request merged**